### PR TITLE
Fix horizontal scrolling on mobile

### PR DESCRIPTION
**Pull request by juvirez**
Here is a screenshot of the problem: http://take.ms/Nhup7

### CODE REVIEW

File: site/css/ocamlorg.css [hunk 1]
```diff
@@ -478,8 +478,9 @@ ul.translations:after { content: "]"; }
 }
 @media (max-width: 480px) {
   #footer .column {
-    width: 98%;
+    width: 80%;
     padding-left: 20%;
+    margin-right: 0;
```
Comment (pw374) [inline site/css/ocamlorg.css]: I think that a better fix would be to remove the `padding-left` completely and add `margin-right: 0` without touching `width`.

**Review by pw374 — CHANGES_REQUESTED** [review]
I think that a better fix would be to remove the `padding-left` completely and add `margin-right: 0` without touching `width`. Please let me know what you think.

**Comment by pw374** [comment]
@juvirez

**Comment by avsm** [comment]
@patricoferris @kanishka-work would either of you be able to look into this one?

**Comment by patricoferris** [comment]
Sure, it also falls under some other "responsive" bugs I've found such as this `iframe` :))

video-width

**Comment by avsm** [comment]
Fixed by #1210

**Pull request closed without merging**